django/db/models/expressions.py

```diff
@@ -140,6 +140,12 @@ class BaseExpression:
     is_summary = False
     _output_field_resolved_to_none = False
 
+    # If the expression can be used in a WHERE-clause.
+    filterable = True
+
```

Comment:
I wouldn't include blank lines between the attributes since they already have comments separating them.